docs/tool/js/views/project-view.js

```diff
@@ -222,25 +222,29 @@ var projectView = {
       }
     },
     location: {
-      title:'Location Information',
-      wrapperPartial: 'partials/project-view/location.html',
-      hideTitle: true,
-      render: function(full_project_data){
-          var data = [];
-          data.push({title:'Ward',value: full_project_data['ward']})
-          data.push({title:'Neighborhood Cluster',value: (full_project_data['neighborhood_cluster'] + ": " + full_project_data['neighborhood_cluster_desc'])})
-          data.push({title:'ANC',value: full_project_data['anc']})
-          data.push({title: 'Census Tract',value: full_project_data['census_tract']})
-
-
-          var table = new D3Table('#location-table')
-            .data(data)
-            .columns([
-                {field:'title', label:'Title', class:'title', html: function(d){return d}},
-                'value'])
-            .hideTitle(true)
-            .create()
-      }
+        title: 'Location Information',
+        wrapperPartial: 'partials/project-view/location.html',
+        hideTitle: true,
+        render: function(full_project_data){
+            var data = [];
+            data.push({title:'Ward',value: full_project_data['ward']})
+            data.push({title: 'Neighborhood Cluster',value: (full_project_data['neighborhood_cluster'] + ": " + full_project_data['neighborhood_cluster_desc'])})
+            data.push({title: 'ANC', value: full_project_data['anc']})
+            data.push({title: 'Census Tract', value: full_project_data['census_tract']})
+
+            var table = new D3Table('#location-table')
+                .data(data)
+                .columns([
+                    {field:'title', label:'Title', class:'title', html: function(d){return d}},
+                    'value'])
+                .hideTitle(true)
+                .create();
+
+            var projLongitude = full_project_data['longitude'];
+            var projLatitude = full_project_data['latitude'];
+
+            d3.select('#project-location-map').attr('src', 'https://api.mapbox.com/styles/v1/mapbox/light-v9/static/pin-s-star+bd3633(' + projLongitude + ',' + projLatitude + ')/-77.0369,38.9072,8.3/124x124?access_token=' + mapboxgl.accessToken + '&attribution=false&logo=false');
```

Comment:
I couldn't figure out how to implement a simple elliptical marker, so this is using the pin marker. I tried setting up a simple GeoJSON to see if it would provide more options for the marker style, but no luck.

Reply:
Pin marker looks great - we'll go with that! 

I might consider making the pin slightly smaller if that's easy, but also fine as is.